### ISSUE TITLE
Console and HTML coverage reports differ in branch miss details

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#342]
*Originally reported by* **Rowan Pattieson (Bitbucket: [rowanpattieson-ipl](https://bitbucket.org/rowanpattieson-ipl), GitHub: Unknown)**

----------------------------------------

Using coverage 3.7.1 .

The "BrMiss" field in a console report and the "partial" field in an HTML report sometimes do not agree with one another.

A simple example to demonstrate this (borrowed from [Issue #176](#176/report-incorrect-with-branch-coverage)) is attached as DifferentBranchMissReport.py .

Notice that the console report states there are 16/16 branch misses, whereas the HTML report states there are 0/16 branch misses.

I think the HTML report is correct.
(None of the branches are *ever* executed, thus they are counted as full misses rather than partial branch misses.) 

Also attached are the coverage output, and console and HTML reports.

Commands executed :

```
#!bash

coverage run --branch DifferentBranchMissReport.py
coverage report > DifferentBranchMissReport_Console.txt
coverage html
```

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/342
- This issue had attachments: .coverage, DifferentBranchMissReport.py, DifferentBranchMissReport_Console.txt, htmlcov.zip.  See the [original issue](https://bitbucket.org/ned/coveragepy/issue/342) for details.

**Comment by nedbat** [comment]
Thanks for the detailed report.  This issue was fixed last month in changeset [bccc40e06c24404e769ef5a5db279f73dadf5113 (bb)](https://bitbucket.org/ned/coveragepy/commits/bccc40e06c24404e769ef5a5db279f73dadf5113), though I had neglected to mention it in the CHANGES.txt file. I've updated the CHANGES file in [96aed4a8786d (bb)](https://bitbucket.org/ned/coveragepy/commits/96aed4a8786d)

**Comment by nedbat** [comment]
*Original comment by* **Rowan Pattieson (Bitbucket: [rowanpattieson-ipl](https://bitbucket.org/rowanpattieson-ipl), GitHub: Unknown)**

----------------------------------------

Great - didn't manage to discover that it was already fixed in the latest codebase  :)
